### PR TITLE
Hw lottery solution

### PR DESCRIPTION
**Pull request by mkazarian-1**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:8]: image

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[random.nextInt(Colors.values().length)].toString();
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:9]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected

File: src/main/java/core/basesyntax/Application.java (hunk 1 above)
Comment (mkazarian-1) [reply src/main/java/core/basesyntax/Application.java:8]: In this case, I thought it was not necessary for the demonstration

File: src/main/java/core/basesyntax/ColorSupplier.java (hunk 2 above)
Comment (mkazarian-1) [reply src/main/java/core/basesyntax/ColorSupplier.java:9]: In this case, I forgot about the existence of this command, so sorry

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), random.nextInt(101));
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/Lottery.java:9]: No need to instantiate `ColorSupplier` each time after calling the method `getRandomBall()`.

You can instantiate it once as you did with `Random`.

Comment (sokimaaa) [inline src/main/java/core/basesyntax/Lottery.java:9]: let's avoid using magic numbers. 

see:
All magic numbers in your code should be constants.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

**Review by sokimaaa — CHANGES_REQUESTED** [review]

**Review by aaananas — APPROVED** [review]

**Pull request closed without merging**